modules/server/src/test/ts/compiler/ImportsTest.ts

```diff
@@ -63,13 +63,38 @@ addTest("/${filename}.js");`
         filenames.forEach((filename) => {
           assert.include(imports, `
 __currentTestFile = "/${filename}.js";
-import "${filename}";
+import "${filename}.js";
 addTest("/${filename}.js");`
           );
         });
       });
     });
   });
+  context('ESM', () => {
```

Comment:
```suggestion

  context('ESM', () => {
```
Newline for better readability